arrays/arrays.c

```diff
@@ -49,14 +49,19 @@ void destroy_array(Array *arr)
  *****/
 void resize_array(Array *arr)
 {
-
   // Create a new element storage with double capacity
-
+  int newStorage = arr->capacity;
+  char **newElement = calloc(newStorage * 2, sizeof(char *));
```

Comment:
I love your use of calloc, in my opinion in terms of efficiency, calloc is the way to go in terms of allocating